components/accountability/AccountabilityHeatmap.tsx

```diff
@@ -0,0 +1,231 @@
+import {
+  eachDayOfInterval,
+  endOfYear,
+  format,
+  getDay,
+  startOfWeek,
+  startOfYear,
+} from "date-fns";
+import React, { useMemo } from "react";
+
+interface Entry {
+  creationDate: string;
+  isCompleted: boolean | null;
+}
+
+interface AccountabilityHeatmapProps {
+  entries: Entry[];
+  year?: number;
+  cellSize?: number;
+  cellGap?: number;
+  colors?: {
+    completed: string;
+    notCompleted: string;
+    noData: string;
+    empty: string;
+  };
+}
+
+const AccountabilityHeatmap: React.FC<AccountabilityHeatmapProps> = ({
+  entries,
+  year = new Date().getFullYear(),
+  cellSize = 16,
+  cellGap = 2,
+  colors = {
+    completed: "rgb(241, 135, 1)", // Skillfy Orange
```

Comment:
these shouldn't be hard-coded. you should be getting these colours from our tailwind theme. 

if the colours don't exist in our theme then add it there to tailwind.config.js

the orange is `charmander`